config/prod/s3-synapse-sync.yaml

```diff
@@ -1,6 +1,6 @@
 template:
   type: "http"
-  url:  "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/s3-synapse-sync/v1.6.1/s3-synapse-sync.yaml"
+  url:  "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/s3-synapse-sync/v1.7.2/s3-synapse-sync.yaml"
```

Comment:
Should it be 1.7.2 (from tags in project)?

Reply:
oh whoops, good catch, I hadn't noticed that the tag format changed